src/Command/CertificationRequestRecomputeCommand.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Command;
+
+use App\Adherent\Certification\CertificationRequestProcessCommand;
+use App\Entity\CertificationRequest;
+use App\Repository\CertificationRequestRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+class CertificationRequestRecomputeCommand extends Command
+{
+    protected static $defaultName = 'app:certification-request:recompute';
+
+    private $certificationRequestRepository;
+    private $em;
+    private $bus;
+    /** @var SymfonyStyle */
+    private $io;
+
+    public function __construct(
+        CertificationRequestRepository $certificationRequestRepository,
+        EntityManagerInterface $em,
+        MessageBusInterface $bus
+    ) {
+        $this->certificationRequestRepository = $certificationRequestRepository;
+        $this->em = $em;
+        $this->bus = $bus;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->addOption('date', null, InputOption::VALUE_REQUIRED, 'Interval in hours (format: YYYY-MM-DD)', 'now')
```

Comment:
`Interval in hours (format: YYYY-MM-DD)` => `Certification created date (format: YYYY-MM-DD, default: now)`